proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -384,52 +378,79 @@ message DispatchUpdateRequest {
       repeated uint32 bymonths = 8;
     }
 
-    // The type of dispatch
-    optional string type = 1;
-
     // The start time
     // When updating a dispatch, ensure that the starting timestamp is set to
     // the current time or any future time.
     // Timestamps earlier than the current time are not allowed.
-    google.protobuf.Timestamp start_time = 2;
+    google.protobuf.Timestamp start_time = 1;
 
     // Duration in seconds
-    optional uint32 duration = 3;
+    optional uint32 duration = 2;
 
     // The component selector
-    ComponentSelector selector = 4;
+    ComponentSelector selector = 3;
 
     // The "active" status
-    optional bool is_active = 5;
-
-    // The "dry run" status
-    optional bool is_dry_run = 6;
+    optional bool is_active = 4;
 
     // The dispatch payload
-    google.protobuf.Struct payload = 7;
+    google.protobuf.Struct payload = 5;
 
     // The recurrence rule
-    RecurrenceRuleUpdate recurrence = 8;
+    RecurrenceRuleUpdate recurrence = 6;
   }
 
+  // ID of the microgrid
+  uint64 microgrid_id = 1;
+
   // The dispatch identifier
-  uint64 id = 1;
+  uint64 dispatch_id = 2;
 
   // Field mask specifying which fields should be updated
-  google.protobuf.FieldMask update_mask = 2;
+  google.protobuf.FieldMask update_mask = 3;
 
   // The updated dispatch attributes
-  DispatchUpdate update = 3;
+  DispatchUpdate update = 4;
+}
+
+// Response message for updating a dispatch
+message UpdateMicrogridDispatchResponse {
+  // The updated dispatch
+  DispatchDetail dispatch = 1;
```

Comment:
Maybe `details`? Otherwise accessing the dispatch will be `response.dispatch.dispatch`.

Reply:
Yeah, I was also fearing this name-space repetition pollution.. 

https://github.com/frequenz-floss/frequenz-api-electricity-trading/blob/v0.x.x/proto/frequenz/api/electricity_trading/v1/electricity_trading.proto#L613

is using a combination of your suggestion and this:
```
  // Details of the updated order.
  OrderDetail order_detail = 2;
```